stl/inc/flat_map

```diff
@@ -1328,6 +1329,7 @@ public:
 
 private:
     using _MyBase::_Erase_if;
+    using _MyBase::_Insert_exact

```

Comment:
Clang-format mis-suggested. The real error is the missing of a `;`.
```suggestion
    using _MyBase::_Insert_exact;
```

Reply:
Haha, yeah I noticed. Fixed now :)